code/modules/ghostroles/spawner/human/human.dm

```diff
@@ -14,6 +14,7 @@
 	var/list/species_outfits = list() //Outfit overwrite for the species
 	var/uses_species_whitelist = TRUE //Do you need the whitelist to play the species?
 	var/possible_species = list("Human")
+	var/default_species = "Human"
```

Comment:
Could just take the first entry of possible_species instead of adding a extra var

Reply:
Good idea, it is done.